static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -165,10 +165,14 @@
    ========================================================================== */
 
 .caption {
-    @include fs-textSans(1);
+    @include fs-textSans(2);
```

Comment:
Looks good, I'll get onto the baselining today

On 19 July 2016 at 10:20, Gideon Goldberg <notifications@github.com> wrote:

> In static/src/stylesheets/module/external/_from-content-api.scss
> <https://github.com/guardian/frontend/pull/13640#discussion_r71300695>:
>
> > @@ -165,10 +165,14 @@
> >     ========================================================================== */
> >
> >  .caption {
> > -    @include fs-textSans(1);
> > +    @include fs-textSans(2);
>
> Argh, good spot that is a mistake, will remove. ❎
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/guardian/frontend/pull/13640/files/9d376b53e2e24a2904f3f920e187c0581ec3c453#r71300695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIgOsdn5D5UpsH02hBCjcIlhQyhRmp7Bks5qXJbegaJpZM4JPhOS>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396